package.json

```
@@ -2,10 +2,12 @@
   "name": "verto-app",
   "version": "0.0.1",
   "scripts": {
-    "dev": "sapper dev",
-    "build": "sapper build --legacy",
-    "export": "sapper export --legacy --entry \"/ trade tokens post login gallery app 404 app/all-exchanges app/all-transactions\"",
-    "start": "node __sapper__/build",
+    "dev": "run-p routify nollup",
+    "build": "run-s build:*",
+    "build:app": "routify -b && rollup -c",
+    "build:static": "spank",
```

to make the CI and vercel happy:
```suggestion
    "export": "spank",
```

---

The vercel config will have to be updated anyways tho

---

hmm... yeah